src/stores/ProjectMemberGroupsStore.js

```diff
@@ -35,6 +43,27 @@ export const useProjectMemberGroupsStore = defineStore({
       }
       return false
     },
+    async editGroup(projectId, groupId, changes) {
+      const url = `${
+        import.meta.env.VITE_API_URL
+      }/projects/${projectId}/groups/${groupId}/edit`
+      const response = await axios.post(url, {
+        group_id: groupId,
+        group_name: changes.group_name,
+        description: changes.description,
+      })
+      const group = response.data.group
+      if (response.status == 200) {
+        this.updateGroup(group.group_id, group.group_name, group.description)
```

Comment:
I did it so that when the groups are listed it doesn't go to the bottom of the list and still maintains the order of the groups but I'll add it that way though.